src/components/HomePage/WhyWCC.js

```diff
@@ -12,8 +12,8 @@ function WhyWCC() {
 			</p>
 			<div>
 				{/* Point 1 */}
-				<div className="flex justify-center items-center py-32">
```

Comment:
we will just create a array for it and map through it cuz that way we can just keep one div

Reply:
Will get to it after this gets merged. I'm not trying to rebase the array change with this responsive stuff 😢

Reply:
ok if u can fix the color, should be good to go then